src/commonMain/kotlin/com/eiknat/lemmyclient/api/site/response/GetSiteResponse.kt

```diff
@@ -17,6 +15,7 @@ data class GetSiteResponse(
     val online: Int,
     val version: String,
     @SerialName("my_user")
+    @Contextual
```

Comment:
this will make it stop complaining but won't actually work until the data class is added. jfyi no big deal tho

Reply:
Yea, that's why I expected. I'll leave a todo so we don't forget